Node.js/GettingStarted/processcsv.py

```diff
@@ -0,0 +1,100 @@
+import csv
+import json
+from azure.storage.blob import BlockBlobService
+import datetime
+import os
+import argparse
+
+#### TODO: Get the configuration
+
+storage_acc_name = '<storage account name>'
+storage_acc_key = '<storage account key>'
+block_blob_service = BlockBlobService(account_name=storage_acc_name,account_key=storage_acc_key)
+####
+
+def getfilename(name):
+
+    names = str(name).split('/')
+    return names[len(names)-1]
+
+
+
+def processcsvfile(fname,seperator,outdir,outfname):    
+    header = []    
+    with open(fname) as inpfile:
+        
+        allrows = csv.reader(inpfile,delimiter=seperator)
+        print("loaded file " + fname)
+        all_vals = []
+
+        for rows in allrows:
+            if isHeaderLoaded is False:
+                header = rows
+            else:
+                if len(header) > 0:
+                    i = 0
+                    line = "{"
+                    
+                    for r in rows:
+                        if i == 0:
+                            
+                            line = line + '"' + header[i] + '":"' + r.decode('utf-8','ignore') + '"'
+                        else:
+                            
+                            line = line + "," + '"' + header[i] + '":"' + r.decode('utf-8','ignore') + '"'
+                        i = i + 1
+
+                    line = line + "}"            
+                    
+
+            all_vals.append(json.loads(json.dumps(line)))
```

Comment:
I'm beginning to feel lost.  Some well-named helper functions, or comments explaining what this method does and how it works, would be really helpful here.